catalyst/README.md

```diff
@@ -50,101 +53,103 @@ By default, instrumentDOM prefixes the classname with `evolv-` and uses the prop
 Each element is also tagged with a unique attribute to handle idenpotency (prevent an element from being manipulated more than once).
 */
 store.instrumentDOM({
-deviceTile:{
-		get dom() {
-				return $('.device-tile, .byod-device-tile');
-		}
-},
-podParent:{
-		get dom() {
-				return $('.evolv-deviceTile [id*=mvo_ovr_devices]').first().parent();
-		}
-},
-buttonParent:{
-		get dom(){return $('button.addALine)').parent();},
-		asClass: 'my-unique-button-class' // optional. 
-}
+  deviceTile:{
+    get dom() {
+      return $('.device-tile, .byod-device-tile');
+    }
+  },
+  podParent:{
+    get dom() {
+      return $('.evolv-deviceTile [id*=mvo_ovr_devices]').first().parent();
+    }
+  },
+  buttonParent:{
+    get dom(){ 
+      return $('button.addALine)').parent(); 
+    },
+    asClass: 'my-unique-button-class' // optional. 
+  }
 });
 ```
 
 Variant level coding
 
-```
+```js
```

Comment:
I'd break these code blocks out separately by language and use `h4` headings instead of the `/**** Comment Headings *****/` inside the blocks

Reply:
fixed